rim/views.py

```diff
@@ -126,3 +143,13 @@ class EditEquipmentView(UpdateView):
     model = Equipment
     form_class = EquipmentForm
     success_url = reverse_lazy('home')
+
+class ClientView(DetailView):
+    template_name = 'rim/client.html'
+    model = Client
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(ClientView, self).get_context_data(*args, **kwargs)
+        context['active'] = Client.objects.get(pk=self.kwargs['pk']).checkout_set.filter(equipment__latest_checkout__pk=F('pk'))
+        context['previous'] = Client.objects.get(pk=self.kwargs['pk']).checkout_set.exclude(equipment__latest_checkout__pk=F('pk'))
```

Comment:
In order to DRY things out on these two lines, you should probably use `context['client']` instead of `Client.objects.get(pk=self.kwargs['pk'])`

Reply:
fixed